src/org/pahappa/systems/ticketing/models/Ticket.java

```diff
@@ -1,11 +1,34 @@
 package org.pahappa.systems.ticketing.models;
 
+import org.pahappa.systems.ticketing.constants.TicketStatus;
+
 /**
  * A ticket refers to a unit of work or a request that is submitted
- * by a user or customer to seek assistance, report an issue, or request a service.
- * It serves as a record or a container that captures all the relevant information
+ * by a user or customer to seek assistance, report an issue, or request a
+ * service.
+ * It serves as a record or a container that captures all the relevant
+ * information
  * related to the user's request or issue.
  */
 public class Ticket {
+    public String Client;
+    public String Contact;
```

Comment:
For variable naming try using java recommended convention.

`https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html`